nemo/core/classes/common.py

```diff
@@ -334,3 +344,8 @@ def __call__(self, wrapped, instance: Typing, args, kwargs):
         instance._attach_and_validate_output_types(outputs)
 
         return outputs
+
+    @classmethod
+    def set_typecheck_enabled(cls, enabled: bool = True):
```

Comment:
why is this a classmethod? If you're using a global variable, why not make it a staticmethod?

Reply:
Good point. I forgot the difference.

Reply:
Well, thats why I chose classmethod vs staticmethod. Staticmethod requires me to pass `cls`, classmethod doesnt require that.

Reply:
Done.